language/en-GB/tpl_cassiopeia.ini

```diff
@@ -12,8 +12,4 @@ TPL_CASSIOPEIA_STATIC="Static"
 TPL_CASSIOPEIA_STICKY_LABEL="Sticky Header"
 TPL_CASSIOPEIA_TOGGLE="Toggle Navigation"
 TPL_CASSIOPEIA_XML_DESCRIPTION="Continuing the space theme (Solarflare from 1.0, Milkyway from 1.5 and Protostar from 3.0), Cassiopeia is the Joomla 4 site template based on Bootstrap 4."
-TPL_CASSIOPEIA_FONT_LABEL="Google font for text and headings"
-TPL_CASSIOPEIA_FONT_DESC="Load Google font for text and headings (H1, H2, H3, etc)."
-TPL_CASSIOPEIA_FONT_NAME_LABEL="Google Font Name"
-TPL_CASSIOPEIA_FONT_NAME_DESC="Example: only one font Josefin Sans or a font combination Montserrat + Work Sans."
-
+TPL_CASSIOPEIA_FONT_LABEL="Extra font(s) for text and headings"
```

Comment:
What else could a font be for?

Reply:
@brianteeman 

Ok, I'll remove the " for text and headings".

Is the "Extra font(s)" ok? Or should it be somethine else, e.g. "Load font(s)" or "Add font(s)"? And is the "(s)" ok for showing it could be a font or a combination of fonts, depending on the particular (s)css file?

Reply:
To match the style guide it should be capital F

It is not very accessible to a screen reader

Is it a multiple select?

Needs to be clear that this will be for fonts that are not part of a users system. ie they will be downloaded over the internet by every user. Personally I would never use this as it will effect the performance of the site

Reply:
@brianteeman Capitalisation "Font(s)" already fixed.

It is a single select, but it selects a css file. This can use one font for all, or a conbination of fonts, e.g. font x for h1, h2 ... h6 and font y for body, or however you define it there.

Correct but maybe too long would be "Extra Font or Font Combination". Shall I use that?

Reply:
And should we translate the font or font combination names, too? In most languages I've ever seen, e.g. Russian or Japanese, such names are (or at least can be) treated as literals with latin characters. But you have more experience in web, so maybe my experience is wrong because too limited.

Reply:
It would be impossible to translate the name and it would be wrong. Would you expect me to translate Richard or Brian?

Reply:
> Would you expect me to translate Richard or Brian?

No, not really. But maybe transliterate to another writing system.

How about the label,  "Extra Font or Font Combination"? Would that be ok?

Reply:
> It is a single select, but it selects a css file. This can use one font for all, or a conbination of fonts, e.g. font x for h1, h2 ... h6 and font y for body, or however you define it there.

Then it is as clear as mud what the purpose is. As a template designer I might want to offer my users the option of 1/3 serif and 1/4 sans serif fonts. If I was stupid enough to ship the fonts how would I expect the user to select them? Create 144 combinations for them to select from

Reply:
No, create a CSS file which uses e.g. "Font X" for h1 and "Font Y" for the (body), and give this file a telling name in the dropdown, e.g. "Font X + Font Y".

Reply:
"Heading Font 1 + Body Font 1"
"Heading Font 1 + Body Font 2"
"Heading Font 1 + Body Font 3"
"Heading Font 1 + Body Font 4"
"Heading Font 2 + Body Font 1"
"Heading Font 2 + Body Font 2"
"Heading Font 2 + Body Font 3"
"Heading Font 2 + Body Font 4"
etc

Reply:
Yes, that works for the dropdown's options so they need no translation.

How about the label of the dropdown? Shall I use "Extra Font or Font Combination"? Or do you have a better idea?

Reply:
Or just "Extra Font Styling" or something like that?

Reply:
Ah, regarding number of options: Sure we should not combine everything which is possible, we should offer very few fonts or combinations which look good in Cassiopeia.

Reply:
That's why I asked about multiple select.

Honestly this feature makes the UX complicated. I was under the impression that this template was to be a showcase example of best practices etc. This entire local fonts thing is doing the opposite

Reply:
So you won't at least suggest a better label text or tell me "Extra Font or Font Combination" is ok?

Reply:
Its not possible to suggest something because its not clear what the feature does

Extra Font means extra to what
Font combination doesn't indicate that it will be using font files that have a negative impact on my sites performance

Reply:
Check the code and test the PR and you will see what the feaure does.

Reply:
I can see what it does - I cant see what it is intended to do

Reply:
Well I think that was explained before: We want to provide a possibility to use locally stored fronts for those who don't wanna load them from Google fonts for whatever reason, GDPR or whatever else.

It was requested to have that possibility, and it was on the task list for that Cassiopeia working group before I joined it.

If that makes sense or not I don't really want to decide. It was not my idea, I only wanted to help with implementation.

Because the was using a CSS file to load the fonts we can do more than just load one font, and so we can select one CSS file which uses several fonts. Currtently the examples here don't do that.

I wanted to keep this work piece separate so we could later still decide to use it or not, but development here was going the way that it was merged into the development branch.

So later it might be harder to revert that part, but it will still be possible, and I would be the last one who would fight against reverting if it was clear that the feature is not accepted by the majority or even a large part of the community.

You are right regarding performance of loading those fonts from the site. Maybe we should mention that in docs, if that fautre is accepted, or even show a kind of warning text in the template style settings.